gremlin-dotnet/src/Gremlin.Net/Process/Utils.cs

```diff
@@ -63,5 +66,23 @@ public static void Forget(this Task task)
                 t.Exception?.Handle(_ => true);
             }, TaskContinuationOptions.ExecuteSynchronously);
         }
+
+        /// <summary>
+        ///  Returns a user agent for connection request headers.
+        ///
+        /// Format:
+        /// "[Application Name] [GLV Name].[Version] [Language Runtime Version] [OS].[Version] [CPU Architecture]"
+        /// </summary>
+        private static string GenerateUserAgent()
+        {
+            var applicationName = Assembly.GetExecutingAssembly().GetName().Name.Replace(' ', '_');
+            var driverVersion = AssemblyName.GetAssemblyName("Gremlin.Net.dll")?.Version.ToString().Replace(' ', '_');
```

Comment:
`GetAssemblyName()` can't return `null` if I'm not mistaken. It will instead throw if it cannot find the assembly (which should never happen), but `Version` can be `null`. So please change this to:
```suggestion
            var driverVersion = AssemblyName.GetAssemblyName("Gremlin.Net.dll").Version?.ToString().Replace(' ', '_');
```